proplet/service.go

```
@@ -350,23 +295,34 @@ func (p *PropletService) UpdateRegistry(ctx context.Context, registryURL, regist
 	return nil
 }
 
-func (p *PropletService) registryUpdate(ctx context.Context, client mqtt.Client, msg mqtt.Message, logger *slog.Logger) {
-	var payload struct {
-		RegistryURL   string `json:"registry_url"`
-		RegistryToken string `json:"registry_token"`
-	}
-	if err := json.Unmarshal(msg.Payload(), &payload); err != nil {
-		logger.Error("Invalid registry update payload", slog.Any("error", err))
+func (p *PropletService) registryUpdate(ctx context.Context) func(topic string, msg map[string]interface{}) error {
```

golangci-lint (1.62.2)

func `(*PropletService).registryUpdate` is unused

(unused)